src/nidigital/examples/nidigital_burst_with_start_trigger.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+
+import argparse
+import nidigital
+import os
+import sys
+
+
+def example(resource_name, options, source, rising_edge):
+
+    with nidigital.Session(resource_name=resource_name, options=options) as session:
+
+        dir = os.path.dirname(__file__)
+
+        # Configure the session by loading pin map (.pinmap) created in the Digital Pattern Editor on the instrument
+        pin_map_filename = os.path.join(dir, 'burst_with_start_trigger_files', 'pin_map.pinmap')
+        session.load_pin_map(pin_map_filename)
+
+        # Configure the session by loading the specifications (.sepcs), levels (.digilevels), and timing (.digitiming) files created in the Digital Pattern Editor on the instrument
+        spec_filename = os.path.join(dir, 'burst_with_start_trigger_files', 'specifications.specs')
+        levels_filename = os.path.join(dir, 'burst_with_start_trigger_files', 'pin_levels.digilevels')
+        timing_filename = os.path.join(dir, 'burst_with_start_trigger_files', 'timing.digitiming')
+        session.load_specifications_levels_and_timing(spec_filename, levels_filename, timing_filename)
+
+        # Apply the settings from the levels and timing files we just loaded on the instrument.
+        session.apply_levels_and_timing(levels_filename, timing_filename)
+
+        # Configure the session by loading the pattern file (.digipat) created in the Digital Pattern Editor on the instrument.
+        pattern_filename = os.path.join(dir, 'burst_with_start_trigger_files', 'pattern.digipat')
+        session.load_pattern(pattern_filename)
+
+        # Specify a source and edge for the external start trigger
+        session.digital_edge_start_trigger_source = source
+        session.digital_edge_start_trigger_edge = nidigital.DigitalEdge.RISING if rising_edge else nidigital.DigitalEdge.FALLING
```

Comment:
Ideally, we want the examples to run "out of the box". With digital edge triggering, the example would hang waiting for the trigger, right?

Reply:
If the customer is using a real device, then yes, it would wait for a trigger. But with simulation enabled by default, it would pass on simulated device.

But if that is a concern, we can convert the example into using a software start trigger instead.

Reply:
I like the "out of the box" experience to be good on both real and simulated h/w. At the same time, I also like demonstrating how to configure and use external triggering. You could do what we do in LabVIEW and .NET examples - allow both (and also no-trigger case) via input args.

Reply:
Okay, I opt for a simpler example but yeah, I can add input args to switch between start trigger and no start trigger.